content/docs/user-guide/testing/best-practices/_index.md

```diff
@@ -89,14 +89,22 @@ Negative assertions are different from negative test cases. A negative test case
 
 It can be okay to include negative assertions when there is also at least one positive assertion, but they should not be relied on alone. Focus on identifying the expected output.
 
+### Disabling and skipping tests
+
+Ideally, all automated test failures prompt immediately fixing the broken feature, or updating the test when underlying expectations have changed. However, sometimes a tradeoff must be made that breaks a feature to enable other functionality. In a case where functionality is known to be broken, automated tests can be temporarily "skipped" or disabled.
+
+Avoid immediately disabling tests when you see a failure. First consider what action the failure blocks, and how critical it is to unblock. Compare that with the risk tradeoff from all contributors ignoring all future bugs the test could catch.
+
+Whenever a test provided with O3DE gets disabled, please [create an issue](https://github.com/o3de/o3de/issues/new/choose) to track fixing the feature and re-enabling the test.
+
 ## Practices to remember
 
 ### Insanity checks
 
-Ensure a new test is configured to run and report failures by temporarily editing the production code. Intentionally break the code in one way the test should detect. Then, run the test suite. If no failure occurs, investigate why!
+Ensure a new test is configured to run and report failures by temporarily editing the production code. Intentionally break the code in one way that the test should detect, and then run the test suite. If no failure occurs, investigate why!
 
 {{< caution >}}
-Immediately revert any intentional code changes after verifying failure can be detected.
+Immediately revert any intentionally broken code, after verifying failure can be detected by the test.
```

Comment:
Here, "that" is used as the object of the verb "verifying".
```suggestion
Immediately revert any intentionally broken code, after verifying that the failure can be detected by the test.
```

Reply:
I mean to say "that"-clause is the whole object. So "that" is a needed conjunction to the clause "the failure can be detected by the test".